app/queries/get_linkables.rb

```diff
@@ -33,7 +33,7 @@ def call
           state_fallback_order: [:published, :draft]
         )
 
-        Edition.where(id: edition_ids).map do |edition|
+        Edition.with_document.includes(:document).where(id: edition_ids).map do |edition|
```

Comment:
We only need the `includes(:document)` here - the `with_document` does an inner join on documents that we don't use in the condition